test/hotspot/jtreg/vmTestbase/metaspace/stressHierarchy/common/PerformChecksHelper.java

```diff
@@ -136,8 +136,16 @@ private void callMethods(Class<?> clazz)
                     }
                 }
             }
+        } catch (InvocationTargetException ite) {
+            if (ite.getCause() != null && ite.getCause().getMessage().trim().toLowerCase().contains("Metaspace")) {
```

Comment:
Surely this can never be true as you converted to lowercase and are searching for uppercase 'M'. ??

And the `trim()` serves no purpose that I can see when you are using `contains`.

Reply:
I think I did a bit too much cut/paste without reading it.  fixed now.